doc/mgr/prometheus.rst

```diff
@@ -152,6 +153,13 @@ The metrics take the following form:
     ceph_health_detail{name="OSD_DOWN",severity="HEALTH_WARN"} 1.0
     ceph_health_detail{name="PG_DEGRADED",severity="HEALTH_WARN"} 1.0
 
+The module also maintains an in-memory history of health check stats
```

Comment:
Missing a full stop here. This will render in the same paragraph as the following sentence so it will look like they are a single sentence. Would "statistics" be better than "stats"?

```suggestion
The module also maintains an in-memory history of health check statistics.
```